persistence.js

```diff
@@ -51,7 +60,7 @@ MongoPersistence.prototype._setup = function () {
 
   var that = this
 
-  this._connect(function (err, client) {
+  this._connect(async function (err, client) {
```

Comment:
This should not be an async function, `_connect` is not handling it.

Reply:
Done